src/sentry/issues/json_schemas.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any, Mapping
+
+EVENT_PAYLOAD_SCHEMA: Mapping[str, Any] = {
+    "type": "object",
+    "properties": {
+        # required properties
+        "event_id": {"type": "string", "minLength": 1},
+        "platform": {"type": "string", "minLength": 1},
+        "project_id": {"type": "integer"},
+        "tags": {"type": "object"},
+        "timestamp": {"type": "string", "format": "date-time"},
+        "message_timestamp": {"type": "string", "format": "date-time"},
+        # "title": {"type": "string", "minLength": 1}, leaving this out, for now
+        # non-required properties
+        "breadcrumbs": {"type": "object"},
+        "contexts": {"type": "object"},
+        "dist": {
+            "type": "object",
+            "properties": {
+                "dist": {"type": "string", "minLength": 1},
+                "release": {"type": "string", "minLength": 1},
+            },
+            "required": ["dist", "release"],
+        },
+        "environment": {"type": "string", "minLength": 1},
+        "extra": {"type": "object"},
+        "message": {"type": "object"},
```

Comment:
We can be a bit more explicit here too https://develop.sentry.dev/sdk/event-payloads/message/